CONTRIBUTING.md

```diff
@@ -30,43 +30,36 @@ style configurations are commonly useful. For IntelliJ 14, copy the style to
 Style, and set the Scheme to `GoogleStyle`.
 
-## Maintaining clean commit history
-
-We have few conventions for keeping history clean and making code reviews easier
-for reviewers:
-
-* First line of commit messages should be in format of
-
-  `package-name: summary of change`
-
-  where the summary finishes the sentence: `This commit improves gRPC to ____________.`
-
-  for example:
-
-  `core,netty,interop-testing: add capacitive duractance to turbo encabulators`
-
-* Every time you receive a feedback on your pull request, push changes that
-  address it as a separate one or multiple commits with a descriptive commit
-  message (try avoid using vauge `addressed pr feedback` type of messages).
-
-  Project maintainers are obligated to squash those commits into one when
-  merging.
-
 ## Guidelines for Pull Requests
 How to get your contributions merged smoothly and quickly.
  
 - Create **small PRs** that are narrowly focused on **addressing a single concern**. We often times receive PRs that are trying to fix several things at a time, but only one fix is considered acceptable, nothing gets merged and both author's & review's time is wasted. Create more PRs to address different concerns and everyone will be happy.
  
-- For speculative changes, consider opening an issue and discussing it first. If you are suggesting a behavioral or API change, consider starting with a [gRFC proposal](https://github.com/grpc/proposal). 
- 
-- Provide a good **PR description** as a record of **what** change is being made and **why** it was made. Link to a github issue if it exists.
- 
-- Don't fix code style and formatting unless you are already changing that line to address an issue. PRs with irrelevant changes won't be merged. If you do want to fix formatting or style, do that in a separate PR.
- 
-- Unless your PR is trivial, you should expect there will be reviewer comments that you'll need to address before merging. We expect you to be reasonably responsive to those comments, otherwise the PR will be closed after 2-3 weeks of inactivity.
- 
-- Maintain **clean commit history** and use **meaningful commit messages**. See [maintaining clean commit history](#maintaining-clean-commit-history) for details.
- 
+- For speculative changes, consider opening an issue and discussing it to avoid
+  wasting time on an inappropriate approach. If you are suggesting a behavioral
+  or API change, consider starting with a [gRFC
+  proposal](https://github.com/grpc/proposal).
+
+- Follow [typical Git commit message](https://cbea.ms/git-commit/#seven-rules)
+  structure. Have a good **commit description** as a record of **what** and
+  **why** the change is being made. Link to a GitHub issue if it exists. The
+  commit description makes a good PR description and is auto-copied by GitHub if
+  you have a single commit when creating the PR.
+
+  If your change is mostly for a single module (other module changes are
```

Comment:
adding `i.e.` or something similar to `other module changes` would make it clear that it is a clarification and read more easily.

Reply:
I had noticed it wasn't great as well, but then said "meh" to myself. I do think `i.e.` would help a lot, although `e.g.,` is more accurate. So I'll use that.